VideoLocker/res/layout/activity_login.xml

```diff
@@ -59,47 +59,74 @@
                         android:paddingLeft="44dp"
                         android:paddingRight="44dp">
 
-                        <EditText
-                            android:id="@+id/email_et"
-                            style="@style/edit_text_style"
-                            android:hint="@string/email_username"
-                            android:inputType="textEmailAddress"
-                            android:maxLength="100" />
-
-                        <EditText
-                            android:id="@+id/password_et"
-                            style="@style/edit_text_style"
-                            android:layout_marginTop="8dp"
-                            android:hint="@string/password"
-                            android:inputType="textPassword"
-                            android:maxLength="100" />
+                        <android.support.design.widget.TextInputLayout
+                            android:id="@+id/usernameWrapper"
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:paddingStart="-4dp"
+                            android:paddingEnd="-4dp"
+                            android:paddingLeft="-4dp"
+                            android:paddingRight="-4dp"
```

Comment:
Yeah, it is. Negative margins have unspecified behaviour, though they work just fine in LinearLayout (and RelativeLayout).

I think this is fine. Otherwise, we'd have to either replace the background drawable for EditText (which has 4dp padding built in) or add 4dp to our other widgets to compensate.